Mathlib/LinearAlgebra/Charpoly/ToMatrix.lean

```diff
@@ -22,9 +22,9 @@ of `f` in any basis.
 
 universe u v w
 
-variable {R : Type u} {M : Type v} [CommRing R] [Nontrivial R]
-
-variable [AddCommGroup M] [Module R M] [Module.Free R M] [Module.Finite R M] (f : M →ₗ[R] M)
+variable {R : Type u} {M : Type v} {M₁ M₂ : Type*} [CommRing R] [Nontrivial R]
```

Comment:
I think we can be consistent about letting Lean manage universes without any other changes.
```suggestion
variable {R M M₁ M₂ : Type*} [CommRing R] [Nontrivial R]
```